code/2_classification/02_train_models/train_models.py

```diff
@@ -162,6 +157,15 @@ def parse_args():
 
     args = parser.parse_args()
 
+    if args.training_site_sets == None:
```

Comment:
Nitpick: `is None` is the preferred style according to [PEP8](https://peps.python.org/pep-0008/#programming-recommendations).